### PR TITLE
Add WebXR Lighting Estimation reference docs - pt 2

### PR DESCRIPTION
**Pull request by Elchi3**
Part 2 adds reference docs for the `XRLightProbe` object and `XRWebGLBinding.getReflectionCubeMap()`

I haven't documented events in a while, but I made pages for the event itself and the event handler property. Will follow-up with BCD later.

### CODE REVIEW

File: files/en-us/web/api/xrlightprobe/reflectionchange_event/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+<table class="properties">
+  <tbody>
+    <tr>
+      <th>Cancelable</th>
+      <td>No</td>
+    </tr>
+    <tr>
+      <th>Interface</th>
+      <td>{{domxref("Event")}}</td>
```
Comment (jpmedley) [inline files/en-us/web/api/xrlightprobe/reflectionchange_event/index.md:28]: Shouldn't this be `ReflectionChange`?

File: files/en-us/web/api/xrwebglbinding/getreflectioncubemap/index.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  - AR
+  - XR
+  - WebXR
+browser-compat: api.XRWebGLBinding.getReflectionCubeMap
+---
+{{APIRef("WebXR Device API")}}
+
+The **`getReflectionCubeMap()`** method of the {{domxref("XRWebGLBinding")}} interface returns a {{domxref("WebGLTexture")}} object containing a reflection cube map texture.
+
+The texture format is specified by the session's `reflectionFormat`. See the `options` parameter on {{domxref("XRSession.requestLightProbe()")}} and {{domxref("XRSession.preferredReflectionFormat")}} for more details. By default, the `srgba8` format is used. When using a `rgba16f` format, you need to be within a WebGL 2.0 context or enable the {{domxref("OES_texture_half_float")}} extension within WebGL 1.0 contexts.
```
Comment (jpmedley) [inline files/en-us/web/api/xrwebglbinding/getreflectioncubemap/index.md:17]: Neither `XRSession.requestLightProbe()` or `XRSession.preferredReflectionFormat` isn't documented. Is it in one of your other PRs?

File: files/en-us/web/api/xrwebglbinding/getreflectioncubemap/index.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+## Syntax
+
+```js
+getReflectionCubeMap(lightProbe)
+```
+
+### Parameters
+
+- `lightProbe`
+  - : An {{domxref("XRLightProbe")}} object obtained from the session.
```
Comment (jpmedley) [inline files/en-us/web/api/xrwebglbinding/getreflectioncubemap/index.md:28]:
```suggestion
  - : An {{domxref("XRLightProbe")}} object returned by calling {{domxref("XRSession.requestLightProbe()")}}.
```

File: files/en-us/web/api/xrwebglbinding/getreflectioncubemap/index.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+### Return value
+
+A {{domxref("WebGLTexture")}} object.
+
+## Examples
+
```
Comment (jpmedley) [inline files/en-us/web/api/xrwebglbinding/getreflectioncubemap/index.md:35]: Please provide a description of what this example is doing.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/xrwebglbinding/getreflectioncubemap/index.md (hunk 2 above)
Comment (Elchi3) [reply files/en-us/web/api/xrwebglbinding/getreflectioncubemap/index.md:17]: These two were documented in https://github.com/mdn/content/pull/7965 (you reviewed it yesterday).

File: files/en-us/web/api/xrlightprobe/reflectionchange_event/index.md (hunk 1 above)
Comment (Elchi3) [reply files/en-us/web/api/xrlightprobe/reflectionchange_event/index.md:28]: The [spec for `reflectionchange`](https://immersive-web.github.io/lighting-estimation/#event-types) defines "The event MUST be of type Event."

File: files/en-us/web/api/xrlightprobe/onreflectionchange/index.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+{{APIRef("WebXR Device API")}}
+
+The **`onreflectionchange`** property of the {{DOMxRef("XRLightProbe")}} interface is and event handler property for the {{domxref("XRLightProbe.reflectionchange_event", "reflectionchange")}} event.
+
+### Value
+
+A function to be invoked whenever the {{domxref("XRLightProbe")}} receives a {{domxref("XRLightProbe.reflectionchange_event", "reflectionchange")}} event.
+
+## Examples
+
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply files/en-us/web/api/xrlightprobe/onreflectionchange/index.md:26]: Yes, absolutely. Added.

File: files/en-us/web/api/xrlightprobe/onreflectionchange/index.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+```js
+lightProbe.onreflectionchange = event => {
+  glCubeMap = glBinding.getReflectionCubeMap(lightProbe);
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply files/en-us/web/api/xrlightprobe/onreflectionchange/index.md:33]: It is very common to name the WebGL context `gl` and the binding `glBinding`. Most WebXR samples I've encountered so far do this. I've added `const glBinding = new XRWebGLBinding(xrSession, gl);` above to make it extra clear, though.

File: files/en-us/web/api/xrwebglbinding/getreflectioncubemap/index.md (hunk 4 above)
Comment (Elchi3) [reply files/en-us/web/api/xrwebglbinding/getreflectioncubemap/index.md:35]: Added.

File: files/en-us/web/api/xrlightprobe/index.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+None.
+
+## Events
+
+- {{domxref("XRLightProbe.reflectionchange_event", "reflectionchange")}}
+  - : Fired each time the estimated reflection cube map has changed (happens when the user moves around and the environment's lighting changes).
+
+## Examples
+
+Typical use cases of the `XRLightProbe` interface:
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply files/en-us/web/api/xrlightprobe/index.md]: Thanks, I added descriptive headings and descriptions. It is indeed much better.

**Comment by Elchi3** [comment]
Addressed all comments. PTAL

**Review by jpmedley — APPROVED** [review]

**Pull request merged**